map/traffic_manager.cpp

```diff
@@ -315,17 +319,22 @@ void TrafficManager::OnTrafficDataResponse(traffic::TrafficInfo const & info)
 
   // Update cache.
   size_t constexpr kElementSize = sizeof(traffic::TrafficInfo::RoadSegmentId) + sizeof(traffic::SpeedGroup);
-  size_t const dataSize = info.GetColoring().size() * kElementSize;
+
+  // Note. It's necessary to multiply by two because routing and rendering use separate caches.
+  size_t const dataSize = 2 * info.GetColoring().size() * kElementSize;
```

Comment:
BTW, why do you need all this ```2 *``` magic here? Why not to just divide by 2 m_maxCacheSizeBytes in constructor, and leave all computations as is?